VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/individual-participant-status-list/individual-participant-status-list.component.ts

```diff
@@ -67,4 +75,18 @@ export class IndividualParticipantStatusListComponent extends WRParticipantStatu
             );
         }
     }
+
+    private hasUnavailableLinkedParticipants(participant: ParticipantResponse) {
+        if (participant.linked_participants.length) {
+            return participant.linked_participants.some(lp => {
+                const linkedParticipant = this.nonJudgeParticipants.find(p => p.id === lp.linked_id);
+                return (
+                    linkedParticipant &&
+                    linkedParticipant.status !== ParticipantStatus.Available &&
+                    linkedParticipant.status !== ParticipantStatus.InConsultation
```

Comment:
don't think we need the `InConsultation` here. The participant will either be available or not